test/acceptance/pages/Register.scala

```diff
@@ -3,10 +3,14 @@ package acceptance.pages
 import acceptance.util.{TestUser, LoadablePage, Browser, Config}
 
 class Register(val testUser: TestUser) extends LoadablePage with Browser {
-  val url = s"${Config.identityFrontendUrl}/register?skipConfirmation=true"
+  val url = s"${Config.identityFrontendUrl}/register?skipConfirmation=true&returnUrl=${Config.baseUrl}/uk"
 
   def hasLoaded(): Boolean = pageHasElement(createAccountButton)
 
+  def hasCreated(): Boolean = pageHasElement(subscriptionsUKDigital)
```

Comment:
As you say, I guess we might want a more robust way of checking that the Identity account has been created successfully, but this is an improvement on our current status!